docs/index.md

```diff
@@ -13,8 +13,21 @@ Those policies are defined using
 [rego](https://www.openpolicyagent.org/docs/latest/policy-language/) and are
 described here.
 
-Policy Rules
-------------
+Pipeline Policy
+----------------
+
+### Basic Rules
+
+#### `[unexpected_kind]` Check the kind is "PipelineRun"
```

Comment:
Should this be `"Pipeline"` instead of `"PipelineRun"`?

Reply:
Oh yeah, thanks!